### ISSUE TITLE
Cached Banners cause crash when browsing large library being forwarded from flashcard

### ISSUE THREAD

**Issue by TapsofOsiris** [Robz8/TWLoader#466]
### Your 3DS

Tell us which console do you have:

- [ ]  Old 3DS
- [ ]  Old 3DS XL
- [ ]  2DS
- [ ]  New 3DS
- [x]  New 3DS XL
- [ ]  New 2DS XL

### CFW and configuration

Tell us which CFW do you have:

- [x]  Boot9Strap (SigHax)
- [ ]  A9LH (Only Sysnand)
- [ ]  A9LH + Emunand
- [x]  Luma3DS
- [ ]  AuReiNAND
- [ ]  Other CFW [specify]
- [ ]  Older CFW
- [ ]  Not using CFW / Using TWLoader in 3DSX format

### Console version

Tell us which version do you have:

- [x]  >= 11.4
- [ ]  < 11.4

### TWLoader version

Tell us which versions you are using:

- [x]  Latest release
- [ ]  I've compiled myself _[insert here commit hash]_
- [ ]  Old release

### Expected behaviour

Tell us what should happen

Browse Large Library of games (100-ish) from flashcard without crashing.
### Actual behaviour

Tell us what happens instead

Crashes when scrolling through library of games
### Steps to reproduce

1.forward a large library of games from flashcard to TW loader. ( 75 to 100 )
2.open TW loader and scroll through the library
3.Crash will happen in the same spot over and over on my multiple systems

### Stack trace and console log

If you are getting a crash or want to upload your log file, please, write that information between the following quotation marks.

``` I noticed that the crash is tied to the cached banners for the games. Once I deleted the cached banners and launched TW loader, there were no more crashes. I can navigate the game list by the box art and launch the games with just the .ini files, but it would be nice to have working banners for all my games without it crashing my systems.```

**Comment by TapsofOsiris** [comment]
I forwarded my entire flashcard game library to TWloader properly by using the "Add Games" icon in the flashcard side of the loader. But when scrolling through my games I get a crash in the same spot over and over. however, when I delete the contents of the "bnricons" folder my system doesn't crash. Basically I'm stuck browsing and launching my 100+ games, with generic "no info" banner icons, using only the box art and .ini files.

**Comment by TapsofOsiris** [comment]
I haven't updated bootstrap since January. Could that be the cause?

**Comment by Robz8** [comment]
No. bootstrap is only for games on the SD card, and that isn't the cause.
